test/manifold_test.cpp

```diff
@@ -160,6 +160,25 @@ TEST(Manifold, Decompose) {
   RelatedGL(manifolds, input);
 }
 
+TEST(Manifold, DecomposeProps) {
+  std::vector<Manifold> manifoldList;
+  manifoldList.emplace_back(WithPositionColors(Manifold::Tetrahedron()));
+  manifoldList.emplace_back(WithPositionColors(Manifold::Cube()));
+  manifoldList.emplace_back(WithPositionColors(Manifold::Sphere(1, 4)));
+  Manifold manifolds = Manifold::Compose(manifoldList);
+
+  ExpectMeshes(manifolds, {{8, 12, 3}, {6, 8, 3}, {4, 4, 3}});
```

Comment:
This is only testing that `Decompose` has the right number of properties. Are they all zeros? Let's replace this `ExpectMeshes` with an explicit `Decompose` and run `RelatedGL(manifold, input)` on each returned item.

Reply:
How should I handle the order of the output from Decompose(), which I don't think is necessarily the same as the input list?

Reply:
It doesn't matter - you use the whole input list for each one. It'll find the correct one to test against via OriginalID.